safe/utilities/clipper.py

```diff
@@ -246,7 +246,7 @@ def _clip_vector_layer(
 
     writer = QgsVectorFileWriter(
         file_name,
-        'UTF-8',
+        None,
```

Comment:
Please look at this line.

I don't understand why, if we uses 'UTF-8', the new test failed because the unicode character doesn't match.

But if we change it to None, or even ascii, the test succeed.

Well, any feedback here?